Cargo.toml

```diff
@@ -59,6 +59,10 @@ openmls_traits = { git = "https://github.com/wireapp/openmls", rev = "6d2f5a2046
 openmls_basic_credential = { git = "https://github.com/wireapp/openmls", rev = "6d2f5a2046fc12aca602af207e650b94848f09ff", version = "0.2" }
 openmls_x509_credential = { git = "https://github.com/wireapp/openmls", rev = "6d2f5a2046fc12aca602af207e650b94848f09ff", version = "0.2" }
 
+# proteus
+proteus-traits = { package = "proteus-traits", git = "https://github.com/wireapp/proteus", tag = "v2.1.1" }
+proteus-wasm = { package = "proteus-wasm", git = "https://github.com/wireapp/proteus", tag = "v2.1.1" }
```

Comment:
I think the `package = ...` is unnecessary since you're not renaming the package, no? We're using the same names so just
```
proteus-wasm = { git = "https://github.com/wireapp/proteus", tag = "v2.1.1" }
```
should work, since Cargo just uses the requested dependency name (the one on the left side of `=`), right?